dvc/ui/__init__.py

```diff
@@ -260,6 +262,26 @@ def isatty(self) -> bool:
 
         return sys.stdout.isatty()
 
+    def open_browser(self, file: "PathLike"):
```

Comment:
We should also support `str`, so it should really be `str | PathLike`. Let's use [`dvc.types.StrPath`](https://github.com/iterative/dvc/blob/a70b3087b4a2a8d604781a9d4ca34486ff90ee16/dvc/types.py#L6) which does that already.

```suggestion
    def open_browser(self, file: "StrPath"):
```